src/main/resources/hudson/tools/JDKInstaller/config.jelly

```diff
@@ -47,4 +47,11 @@ THE SOFTWARE.
   <f:entry title="" field="acceptLicense">
     <f:checkbox title="${%I agree to the Java SE Development Kit License Agreement}"/>
   </f:entry>
+  <f:entry>
```

Comment:
`<f:entry>` has semantic meaning in terms of form entry and data binding, so it might be better to leave it out and just have the `<div>`, although I don't know how that would look.

Reply:
`<f:entry>`<img width="839" alt="entry" src="https://user-images.githubusercontent.com/304844/58583492-68e1c100-8254-11e9-9a83-00122a9dfc36.png">

`<f:block>`<img width="848" alt="block" src="https://user-images.githubusercontent.com/304844/58583489-68e1c100-8254-11e9-9f81-1d00af40b6fa.png">

Reply:
Using `<f:block>` is just plain html and this should not influence functionality.

Reply:
Ah yeah, I forgot about `<f:block>`. I think it might be better to use that, but I don't feel too strongly. Maybe wait and see what other reviewers think.